app/components/data_table/cell_component.rb

```diff
@@ -0,0 +1,72 @@
+module DataTable
+  class CellComponent < GovukComponent::Base
+    def initialize(colname:, scope:, colspan:, sorting:, numeric:, text:, classes: []) # rubocop:disable Metrics/ParameterLists
```

Comment:
Should this be re-enabled after this method?

Reply:
Comments like this work on per block basis